.github/workflows/pages.yml

```diff
@@ -2,12 +2,12 @@
 name: Deploy Jekyll site to Pages
 
 on:
+  pull_request:
+    branches:
+      - "main"
```

Comment:
Remove this. We want to run this workflow on every PR.

Reply:
Done